libs/classifiers/edgetpu/face_mask.py

```diff
@@ -19,29 +19,13 @@ class Classifier:
 
     def __init__(self, config):
         self.config = config
-        # Get the model name from the config
-        self.model_name = self.config.get_section_dict('Classifier')['Name']
-        # Frames Per Second
+        self.model_name = "OFMClassifier_edgetpu.tflite"
+        self.model_path = 'libs/classifiers/edgetpu/' + self.model_name
```

Comment:
I think it is better to store the models under `/repo/data/...` as then they are on the volume and only need to be downloaded once. Unless there is another reason they have to be in the libs folder